ipn/ipnlocal/local.go

```diff
@@ -2014,7 +2011,18 @@ func (b *LocalBackend) WatchNotifications(ctx context.Context, mask ipn.NotifyWa
 		go b.pollRequestEngineStatus(ctx)
 	}
 
-	defer b.DeleteForegroundSession(sessionID) // TODO(marwan-at-work): check err
+	// TODO(marwan-at-work): check err
+	// TODO(marwan-at-work): streaming background logs?
+	defer b.DeleteForegroundSession(sessionID)
+
+	if mask&ipn.NotifyServeRequest == 0 {
+		fn = func(roNotify *ipn.Notify) (keepGoing bool) {
+			if roNotify.RequestAccessLog != nil {
+				return true
```

Comment:
so your strategy here is to try to filter out non-access log things but otherwise send them all to bus?

that's slightly inefficient when there are no interested listeners at all. it'd be nice to keep a reference count of sessionIDs that care and when it's at zero, don't publish an access log record at all.

Reply:
I thought this would never send messages to uninterested listeners (we can also be more efficient to inverse this and never send anything to interested listeners that don't have a non-nil *RequestAccessLog.) 

In any case, not sure how keeping a reference count would filter out sending those messages to GUIs/uninterested clients?

Reply:
removed sending request logs in the context of this pr